site/docs/extensions/filter-control.md

```diff
@@ -202,7 +192,43 @@ Dependence if you use the datepicker option: [bootstrap-datepicker](https://gith
 
 - **Detail:**
 
-   If the datepicker option is set use this option to configure the datepicker with the native options. Use this way: `data-filter-datepicker-options='{"autoclose":true, "clearBtn": true, "todayHighlight": true}'`.
+   If the datepicker option is set use this option to configure the datepicker with the native options. Use this way: `data-filter-datepicker-options='{"property1":value1, "property2": value2, "property3": value3}'`.
+
+- **Default:** `undefined`
+
+### filterControlMultipleSelect
+
+- **Attribute:** `data-filter-multiple-select`
+
+- **type:** `boolean`
+
+- **Detail:**
+
+   Use this option to configure the select as multipleSelect.
+
+- **Default:** `false`
+
+### filterControlMultipleSelectMultiple
+
+- **Attribute:** `data-filter-multiple-select-multiple`
+
+- **type:** `boolean`
+
+- **Detail:**
+
+   If the `filterControlMultipleSelect` option is set to `true`, use this option to configure the select as multiple select.
+
+- **Default:** `false`
+
+### filterMultipleSelectOptions
```

Comment:
I searched in project for this option, but it seems its never used ?

Reply:
That option is to use the native options from MultipleSelect plugin

Reply:
Sure i understood that, but for me it seems this option is never used while init the multipleSelect plugin (https://github.com/wenzhixin/bootstrap-table/pull/5583/files#diff-82f03929aa03d0669f101189d8e35436371378527780c64a062bb646dd8cd08eR152).
But maybe im wrong :-)

Reply:
Just pull the changes https://github.com/wenzhixin/bootstrap-table/blob/e6f2a51c345b11df3a5ac89b9c1b1bd59901ed79/src/extensions/filter-control/bootstrap-table-filter-control.js#L151

Reply:
Now you pushed it 👍🏻